src/server/identity/db.go

```diff
@@ -16,3 +16,14 @@ CREATE TABLE IF NOT EXISTS identity.users (
 `)
 	return err
 }
+
+// CreateConfigTable sets up the postgres table which stores IDP configuration
+func CreateConfigTable(ctx context.Context, tx *sqlx.Tx) error {
+	_, err := tx.ExecContext(ctx, `
+CREATE TABLE IF NOT EXISTS identity.config (
```

Comment:
I think this lets you remove `IdentityServerConfig` from `src/identity/identity.proto` too?

Reply:
We still use `IdentityServerConfig` in the `(Set|Get)IdentityServerConfig` RPCs.